src/interface/hbase.thrift

```diff
@@ -0,0 +1,622 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// NOTE: The "required" and "optional" keywords for the service methods are purely for documentation
+
+namespace java org.apache.hadoop.hbase.thrift2.generated
```

Comment:
Thanks for your great work 🎉 

The IDL under `interface` was used to define the protocol to communication between components.

It seems better to move the file to another place and take a discussion for more details.

Reply:
You are right, but this hbase.thrift file just copy from [1], do we really want to modify it?  I think that we don't make any changes to it is the best choice. :-)

[1] https://github.com/apache/hbase/blob/rel/1.4.9/hbase-thrift/src/main/resources/org/apache/hadoop/hbase/thrift2/hbase.thrift

Reply:
Yes, agree! I have considered this issue before, and I finally put it here for a while, to see if anyone has a better directory location suggestion? :-)

Reply:
IMO, it should be closer to the place of use.

Reply:
OK, it will be put into src/kvstore/hbase.